andela/bp-esa-backend
# Demo - remove from channel

#### What does this PR do?
Enables users to be removed to channels.

#### Description of Task to be completed?
Placed developers should be removed from the `available-devs` channel when placed.

#### How should this be manually tested?
Assuming you are an administrator of the workspace:

- Create an app. It should have workspace privileges in order to create channels(groups:write)
- Generate a token for the app and add it to the .env file as the SLACK_TOKEN
- Calling the `removeFromChannel` function in server/modules/slackIntegration.js, while providing the user's email and channel ID, will remove user from channel.

#### What are the relevant pivotal tracker stories?
[160436611](https://www.pivotaltracker.com/story/show/160436611)
